Dispatcher/Views/TableViewCells/ArticleCell.xib

```diff
@@ -13,7 +13,7 @@
         <tableViewCell clipsSubviews="YES" contentMode="center" selectionStyle="none" indentationWidth="10" reuseIdentifier="articleCell" rowHeight="460" id="KGk-i7-Jjw" customClass="ArticleCell" customModule="Dispatcher" customModuleProvider="target">
```

Comment:
You have a little issue with the constraints in this file: 
![Screen Shot 2023-05-05 at 14 50 23](https://user-images.githubusercontent.com/100192030/236450391-76470586-ddf0-4d83-9c08-87ab91122001.png)